standalone/src/handlers.rs

```diff
@@ -109,3 +137,94 @@ pub async fn clear_graph() -> Result<Json<ProcessResponse>> {
         )))
     }
 }
+
+pub async fn ingest() -> Result<Json<ProcessResponse>> {
+    let repo_path = env_not_empty("REPO_PATH");
+    let repo_url = env_not_empty("REPO_URL");
+    let username = env_not_empty("USERNAME");
+    let pat = env_not_empty("PAT");
+
+    if repo_path.is_none() && repo_url.is_none() {
+        return Err(AppError::Anyhow(anyhow::anyhow!(
+            "Neither REPO_PATH nor REPO_URL is set in the environment"
+        )));
+    }
+
+    let (final_repo_path, final_repo_url, need_clone) = if let Some(path) = repo_path {
+        (path, repo_url.unwrap_or_default(), false)
+    } else {
+        let url = repo_url.unwrap();
+        let tmp_path = Repo::get_path_from_url(&url)?;
+        (tmp_path, url, true)
+    };
+
+    let username_clone = username.clone();
+    let pat_clone = pat.clone();
+
+    let result = tokio::task::spawn_blocking(move || {
+        #[cfg(feature = "neo4j")]
```

Comment:
i dont think you need to check for neo4j feature here, this `standalone` bin must have neo4j enabled. you can enable it by default in the standalone Cargo.toml